compiler/luci/pass/src/FuseAddWithTConvPass.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright (c) 2020-2024 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
we don't need to do this change

Reply:
I wasn't sure about it as well... now I have guidance to follow. Will do!